R/distance.R

```diff
@@ -0,0 +1,29 @@
+#install.packages('stringdist')
+#install.packages('proxy')
+library(stringdist)
+library(proxy)
+
+
+repDist <- function(.data, .col = "CDR3.nt", .method = "hamming") {
```

Comment:
Function needs header with @importFrom fields, function and parameters description, examples and @export field

Reply:
it is very-very raw draft to prove the concept